### PR TITLE
Enable sending campaigns

### PR DESCRIPTION
**Pull request by ZainRizvi**
Adds the ability to send campaigns after composing them

### CODE REVIEW

File: mailerlite/campaign.py [hunk 1]
```diff
@@ -219,6 +219,25 @@ def create(self, data):
...
+        campaign_id : int
+            campaign id
+
+        Returns
+        -------
+        content : dict
+            The JSON output from the API
+        """
+        url = client.build_url('campaigns', campaign_id, "actions/send")
+        print ("url = " + url)
```
Comment (skoudoro) [inline mailerlite/campaign.py:238]: Can you remove this print?

**Review by skoudoro — COMMENTED** [review]
Hi @ZainRizvi,

Thank you so much for doing this!

Overall, it looks good to me. I just need to try it. 

Can you update the README.rst by adding a really short example/documentation for this new function? See also my comment below. 

Thank you

**Comment by skoudoro** [comment]
Also, Do not forget to address the [pep8 above](https://github.com/skoudoro/mailerlite-api-python/pull/32#issuecomment-916655496)

**Comment by skoudoro** [comment]
Hi @ZainRizvi, 

Your PR is really close to being done and merge. Will you find time to finalize it? Let me know if you have any questions.

**Comment by ZainRizvi** [comment]
Hi @skoudoro, it took me a while to get around to doing this but I've updated the PR :) 

I couldn't find a README.rst, but I figured you meant README.md

**Pull request merged**